### PR TITLE
Fix design issues

### PR DESCRIPTION
**Pull request by AdityaKhatri**
## Changes

* Handle disabled on organigram input
* Fix compact section styling
* Framework detail page styling
* Fix entry cards styling

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

## This PR contains valid:

- [x] permission checks
- [x] translations

### CODE REVIEW

File: app/Base/types/user.ts [hunk 1]
```diff
@@ -2,4 +2,7 @@ export interface User {
     id: string;
     displayName?: string;
     displayPictureUrl?: string;
+    accessibleFeatures?: {
+        key: string;
```
Comment (tnagorra) [inline app/Base/types/user.ts]: The key should have been an ENUM

File: app/components/entry/Section/index.tsx [hunk 2]
```diff
@@ -59,11 +59,15 @@ function Section(props: Props) {
...
-                className={styles.section}
+                className={_cs(
+                    styles.section,
+                    (widgets?.length ?? 0) < 1 && styles.empty,
+                )}
                 data={widgets ?? undefined}
                 keySelector={widgetKeySelector}
                 renderer={AttributeInput}
                 rendererParams={widgetRendererParams}
+                emptyMessage="There are no widgets in this framework"
```
Comment (tnagorra) [inline app/components/entry/Section/index.tsx]: The error message is not exactly correct.
This is a section component

File: app/components/framework/CompactAttributeInput/WidgetWrapper/index.tsx [hunk 3]
```diff
@@ -42,7 +42,7 @@ function WidgetWrapper(props: Props) {
                 className={_cs(headerClassName, styles.header)}
                 // FIXME: use strings
             >
-                {title ?? 'Unnamed'}
+                {title}
```
Comment (tnagorra) [inline app/components/framework/CompactAttributeInput/WidgetWrapper/index.tsx]: Why remove default title?
This is very useful when we are creating a new widget

File: app/views/Project/Tagging/Sources/SourcesGrid/EntryCard/index.tsx [hunk 4]
```diff
@@ -133,6 +133,7 @@ function EntryCard(props: Props) {
                 headerDescription={(
                     <DateOutput
                         value={leadDetails.publishedOn}
+                        format="dd MMM yyyy"
```
Comment (tnagorra) [inline app/views/Project/Tagging/Sources/SourcesGrid/EntryCard/index.tsx]: Do all DateOutput need this format?

File: app/views/ProjectEdit/ProjectDetailsForm/index.tsx [hunk 5]
```diff
@@ -442,6 +445,7 @@ function ProjectDetailsForm(props: Props) {
                             <RequestPrivateProjectButton />
                         )}
                     </Container>
+                    {/*
```
Comment (tnagorra) [inline app/views/ProjectEdit/ProjectDetailsForm/index.tsx:448]: Any reason this is hidden?

File: app/views/ProjectEdit/ProjectDetailsForm/index.tsx [hunk 6]
```diff
@@ -344,9 +339,9 @@ function ProjectDetailsForm(props: Props) {
                     disabled={disabled || pristine}
                     onClick={handleSubmit}
                     variant="primary"
-                    name="saveAndClose"
+                    name="save"
                 >
-                    {_ts('projectEdit', 'saveButtonLabel')}
+                    Save Project
```
Comment (tnagorra) [inline app/views/ProjectEdit/ProjectDetailsForm/index.tsx:344]: Why not just have 'Save'? This is the first place where 'Save' is followed by a noun on a button

File: app/views/ProjectEdit/ProjectDetailsForm/index.tsx [hunk 7]
```diff
@@ -202,6 +202,12 @@ function ProjectDetailsForm(props: Props) {
         setValue,
     } = useForm(schema, initialValue);
 
+    const {
+        user,
+    } = useContext(UserContext);
+
+    const accessPrivateProject = user?.accessibleFeatures?.some((f) => f.key === 'PRIVATE_PROJECT');
```
Comment (tnagorra) [inline app/views/ProjectEdit/ProjectDetailsForm/index.tsx]:
```suggestion
    const accessPrivateProject = !!user?.accessibleFeatures?.find((f) => f.key === 'PRIVATE_PROJECT');
```

Comment (tnagorra) [reply app/views/ProjectEdit/ProjectDetailsForm/index.tsx]: Is the logic already implemented for accessPrivateProject=true condition?

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: app/views/ProjectEdit/ProjectDetailsForm/index.tsx (hunk 5 above)
Comment (AdityaKhatri) [reply app/views/ProjectEdit/ProjectDetailsForm/index.tsx:448]: Only from server now

File: app/views/ProjectEdit/ProjectDetailsForm/index.tsx (hunk 6 above)
Comment (AdityaKhatri) [reply app/views/ProjectEdit/ProjectDetailsForm/index.tsx:344]: Mockups

File: app/views/ProjectEdit/ProjectDetailsForm/index.tsx (hunk 7 above)
Comment (AdityaKhatri) [reply app/views/ProjectEdit/ProjectDetailsForm/index.tsx]: yes

**Pull request merged**